gwcs/schemas/stsci.edu/gwcs/snell3d-1.0.0.yaml

```diff
@@ -0,0 +1,19 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/gwcs/snell3d-1.0.0"
+tag: "tag:stsci.edu:gwcs/snell3d-1.0.0"
+title: Snell Law in 3D space
+
+description: |
+  Snell Law in 3D - inputs and outputs are direction cosines.
+
+allOf:
+  - $ref: "tag:stsci.edu:asdf/transform/transform-1.1.0"
+  - type: object
+    properties:
+      refractive_index:
+        description: |
+          Index of refraction.
```

Comment:
Is number that is all that is permitted? Couldn't it be a model as well that returns a float? For use with a prism, a number is not useful since it should vary with wavelength.